elem.go

```diff
@@ -53,16 +53,21 @@ var booleanAttrs = map[string]struct{}{
 	attrs.Selected:        {},
 }
 
+type RenderOptions struct {
+	DisablePreamble bool
```

Comment:
Can you add this comment above this line please? `// DisablePreamble disables the doctype preamble for the HTML tag if it exists in the rendering tree.`

Reply:
Also, since these options actually are present on any element calling `Render` what do you think about being more specific with a name like `DisableHtmlPreamble`, `DisableDocumentPreamble`, or `DisableDocPreamble`?

Reply:
`DisableHtmlPreamble` looks pretty self-explanatory, as we call it for `html` tag. Also added the comment.